### PR TITLE
Add example config files

### PR DESCRIPTION
**Pull request by marcantoinegodde**
This PR proposes little adjustments and example config files to help during project setup.

### CODE REVIEW

File: ovh/terraform.tfvars.example [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+argocd_password             = "$2a$10$QAlYJhAr2QMETwDeqyEKReXttl3P1BUbMvoukKCL9nZwauYbg0mN6"
+argocd_repo_url             = "https://github.com/example/argocd.git"
+argocd_repo_username        = "argocd"
+argocd_repo_password        = "access_token"
+main_cluster_issuer_name    = "letsencrypt-prod"
+issuers = [
+  {
+    name                    = "letsencrypt-prod"
+    server                  = "https://acme-v02.api.letsencrypt.org/directory"
+    email                   = "",
```
Comment (Quentique) [inline ovh/terraform.tfvars.example]:
```suggestion
    email                   = "yourname@example.com",
```

File: ovh/terraform.tfvars.example [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    private_key_secret_name = "letsencrypt-prod"
+    }, {
+    name                    = "letsencrypt-staging"
+    server                  = "https://acme-staging-v02.api.letsencrypt.org/directory"
+    email                   = ""
```
Comment (Quentique) [inline ovh/terraform.tfvars.example]:
```suggestion
    email                   = "yourname+1@example.com"
```

**Comment by bilelelelelelel** [comment]
We can put the example files in the README of each module, instead of (or in addition to) what is currently there, if we do not want to overload with .example files 🤷‍♂

**Comment by Quentique** [comment]
> We can put the example files in the README of each module, instead of (or in addition to) what is currently there, if we do not want to overload with .example files man_shrugging

Qu'est-ce que t'appelles par module ? Parce qu'actuellement il n'y en a qu'un

Personnellement, je recommenderais de mettre les fichiers `.example` dans le dossier `examples`.

**Pull request merged**